config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+
+  post "/guests/:id", to: "guest_rooms#create"
```

Comment:
I actually would think that a route to `post "/guests/:id"` would create a new guest, except we wouldn't have the :id part, so this route doesn't necessarily follow RESTful conventions. Could have this route be `post "room_guests"` or could do `post "/guests/:id/rooms"`